nextcord/application_command.py

```diff
@@ -0,0 +1,97 @@
+from __future__ import annotations
+from typing import TYPE_CHECKING, Union, Optional, List
+from enum import IntEnum
+from .mixins import Hashable
+from . import utils
+
+if TYPE_CHECKING:
+    from .state import ConnectionState
+    from .guild import Guild
+
+
+class ApplicationCommandType(IntEnum):
+    CHAT_INPUT = 1
+    USER = 2
+    MESSAGE = 3
```

Comment:
These are for Context Menus, right? Maybe name them something like `ContextMenuType`?